SUASSystem/SUASSystem/sda_converter.py

```diff
@@ -171,7 +169,7 @@ def get_distance_to_current_guided_waypoint(self):
         """
         Returns the distance to the current guided waypoint
         """
-        if self.current_path.shape[0] != 0:
+        if does_guided_path_exist:
```

Comment:
This is a great change here. You need to change the call to ```if self.does_guided_path_exist()```, however, since it is a class method call, not a local variable.